src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.cs

```diff
@@ -8,19 +8,8 @@ namespace System.Net.NetworkInformation
 {
     public partial class NetworkChange
     {
-        // The list of current address-changed subscribers.
-        private static readonly Dictionary<NetworkAddressChangedEventHandler, ExecutionContext?> s_addressChangedSubscribers =
-            new Dictionary<NetworkAddressChangedEventHandler, ExecutionContext?>();
-
-        // The list of current availability-changed subscribers.
-        private static readonly Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext?> s_availabilityChangedSubscribers =
-            new Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext?>();
-
         private static readonly NetworkAvailabilityEventArgs s_availableEventArgs = new NetworkAvailabilityEventArgs(isAvailable: true);
         private static readonly NetworkAvailabilityEventArgs s_notAvailableEventArgs = new NetworkAvailabilityEventArgs(isAvailable: false);
-        private static readonly ContextCallback s_runHandlerAvailable = new ContextCallback(RunAvailabilityHandlerAvailable!);
-        private static readonly ContextCallback s_runHandlerNotAvailable = new ContextCallback(RunAvailabilityHandlerNotAvailable!);
-        private static readonly ContextCallback s_runAddressChangedHandler = new ContextCallback(RunAddressChangedHandler!);
```

Comment:
I don't like that this is duplicated now in each file for each supported platform.  I'd rather see whatever is necessary to make the unknown unix implementation compile added to NetworkAddressChange.UnknownUnix.cs.

Reply:
Reverted this change and added a cctor in UnknownUnix with fake usage of these fields.